mypy/semanal_namedtuple.py

```diff
@@ -339,15 +338,13 @@ def parse_namedtuple_args(
         if call.arg_kinds[:2] != [ARG_POS, ARG_POS]:
             self.fail(f'Unexpected arguments to "{type_name}()"', call)
             return None
-        if not isinstance(args[0], (StrExpr, BytesExpr, UnicodeExpr)):
+        if not isinstance(args[0], (StrExpr, BytesExpr)):
```

Comment:
Interesting that we allow bytes names here, but no need to change that in this PR.